VMAccess/vmaccess.py

```diff
@@ -324,7 +331,9 @@ def _reset_sshd_config(sshd_file_path):
                     break
 
         # Prepare cloud init config for coreos-cloudinit
-        cfg_tempfile = "/tmp/cloudinit.cfg"
+        f = tempfile.NamedTemporaryFile(delete=False)
+        f.close()
+        cfg_tempfile = f.name
```

Comment:
It amazes me that coreos-cloudinit has something like 12 options for where to get the data from, and *none of them* is "from stdin". I don't read go well enough to see for myself - if no "data source" option is provided at all, will coreos-cloudinit read from stdin? If so, it would be best to clone the "getting rid of /tmp/kk" change I referenced elsewhere to do the same thing here.